inst/templates/rosepine.R

```diff
@@ -0,0 +1,500 @@
+    ##  SCSS / PARTIALS                                    >>
+    ## >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+    ## -----------------------------------------
+    ##  Command palette                      --
+    ## -----------------------------------------
+    rsthemes::rstheme_command_palette(
+      item_hover_background = "lighten($bg, 5%)",
+      item_selected_background = "$hl_2"
+    ),
+    #
+    ## -----------------------------------------
+    ##  Rainbow parentheses                  --
+    ## -----------------------------------------
+    rsthemes::rstheme_rainbow_parentheses(
+      "$fg_2",
+      "$light_red",
+      "$light_green",
+      "$light_yellow",
+      "$light_blue",
+      "$light_purple",
+      "$light_cyan"
+    ),
+    #
+    ## ----------------------------------------
+    ##  Use large tabs                      --
+    ## ----------------------------------------
+    rsthemes::rstheme_large_tabs(),
+    #
+    ## ----------------------------------------
+    ##  Dialog options                      --
+    ## ----------------------------------------
+    rsthemes::rstheme_dialog_options(
+      background = "$bg_3",
+      heading_foreground = "$rmd_heading_foreground",
+      help_foreground = "$fg_3",
+      border = "$hl_3",
+      selected_background = "$hl_2",
+      button_border = "$hl_3",
+      # button_hover_background = .lighten(
+      #   "ui_rstudio_dialog_button_background",
+      #   2
+      # ),
+      input_border = "$hl_2",
+      input_background = "$bg_3",
+      input_foreground = "$code_string",
+      checkbox_background = "$hl_3",
+      select_background = "$bg_3"
+    ),
+    #
+    ## -----------------------------------------
+    ##  Terminal colors                      --
+    ## -----------------------------------------
+    rsthemes::rstheme_terminal_colors(
+      theme_dark = theme_dark,
+      red = "$red",
+      red_bright = "$light_red",
+      green = "$green",
+      green_bright = "$light_green",
+      yellow = "$yellow",
+      yellow_bright = "$light_yellow",
+      blue = "$blue",
+      blue_bright = "$light_blue",
+      cyan = "$cyan",
+      cyan_bright = "$light_cyan",
+      magenta = "$purple",
+      magenta_bright = "$light_purple",
+      white = "$white",
+      black = "$black"
+    ),
+    #
+    ## -----------------------------------------
+    ##  Flat theme extras                    --
+    ## -----------------------------------------
+    # I used this snippet found in the the `rsthemes` Elm theme code at
+    # https://github.com/gadenbuie/rsthemes inst/templates/elm.R
+    # - add light bar, remove tab outline
+    "
+    .rstudio-themes-flat .gwt-TabLayoutPanelTab-selected {
+      .gwt-TabLayoutPanelTabInner .rstheme_tabLayoutCenter {
+        box-shadow: 0 3px 0 $cyan inset;
+        border-radius: 0 !important;
+
+        .gwt-Label {
+          \\ font-weight: 600;
+          \\ color: $fg;
+        }
+      }
```

Comment:
Let's just delete these lines... 

```suggestion
```

or we need to update the number of slashes (because it's a character string in R)

```suggestion

        .gwt-Label {
          \\\\ font-weight: 600;
          \\\\ color: $fg;
        }
      }
```